Gemfile

```diff
@@ -9,6 +9,7 @@ gem "coffee-rails", "~> 4.1.0"
 gem "clearance"
 gem "delayed_job_active_record"
 gem "email_validator"
+gem "faker", require: false
```

Comment:
Oh, I see in your PR's description. In that case, you want to use the gem on staging environment, but not in production. you could place it in test, development, and staging group. 

```ruby
group :development, :test, :staging do
  gem "faker"
end
```